jui-grid/index.d.ts

```diff
@@ -0,0 +1,996 @@
+// Type definitions for jui-grid 2.0
+// Project: https://github.com/juijs/jui-grid
+// Definitions by: Jin-Ho Park <https://github.com/easylogic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import {UIEvent} from 'jui-core';
+
+export interface GridColumn {
+    (index: number|string): this;
```

Comment:
This is the function call syntax. Given `let x: GridColumn` this would let you do `x(0)`. If you want to be able to do `x[0]`, use `[index: number|string]: this;` instead.